pkg/k8sintrospect/collector.go

```diff
@@ -416,7 +416,7 @@ func (c *collector) Run() error {
 		for _, objCfg := range c.cfg.Objects {
 			objClient := clientMap[objCfg.GVK]
 			if objClient == nil {
-				log.Warningf("Error generating diagnostic bundle. No client found for %s", objCfg.GVK.String())
+				log.Warningf("Missing CRD to collect for generating a diagnostic bundle. No client found for %s.", objCfg.GVK.String())
```

Comment:
It's not given that the reason is a missing CRD. If we had some internal or connectivity error then boldly claiming that CRD is missing for, say, `apps/Deployment` would be at least confusing. How about the following:

```suggestion
				log.Warningf("Missing type information for %q when generating diagnostic bundle. (CRD not present in cluster?)", objCfg.GVK.String())
```

Reply:
That sounds good.